bangazon-clumsy-penguins/BangazonSite
# Arw delete payment

# Name - PR
Delete Payment Type
## Description
##### Provide clarity to any potentially complex code:
Code necessary to allow a user to remove a payment type.

##### Explain reasons behind organizational or architectual decisions you made:
Decided to add an "IsActive" column to the Payment Type table, so that if a user has order history linked to the payment type that (s)he is trying to delete, the payment type would "soft delete" instead by changing the payment type to inactive.

## Steps to Test
git checkout master
git fetch
git checkout arw-deletePayment
git pull
Delete BangazonSite database
Delete Migrations folder
add-migration IsActiveColumn
update-database
Run
Once logged in, click on "Account Settings" in the navbar
Click delete next to a payment type.
Click delete again to confirm
Should be rerouted to the index view and only active payment types should remain

##### System configuration.

##### 3rd party libraries that need to be installed.

##### Command line utilities to run.

##### Link to Feature Ticket
#9
